pkg/machines/scripts/create_machine.sh

```diff
@@ -13,12 +13,25 @@ vmExists(){
 }
 
 endIfFailed(){
-	if [ $1 -ne 0 ]; then
-	    rm -f "$XMLS_FILE"
+    if [ $1 -ne 0 ]; then
+        rm -f "$XMLS_FILE"
         exit $1
     fi
 }
 
+spiceSupported(){
+    # map system architecture to qemu-system-* command
+    ARCH=$(uname -m)
+    case "$ARCH" in
+        i?86) QEMU=qemu-system-i386 ; break ;;
+        ppc64*) QEMU=qemu-system-ppc64 ; break ;;
```

Comment:
I think break is not needed or valid here

Reply:
Eww, indeed; it's a no-op (so I didn't notice it when testing on 32 bit), but it leaves an ugly `break: only meaningful in a `for', `while', or `until' loop` message in the logs. Fixed.